src/containers/Tenant/Query/ExecuteResult/PlanToSvgButton.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import {ArrowUpRightFromSquare} from '@gravity-ui/icons';
+import {Button, Tooltip} from '@gravity-ui/uikit';
+
+import {planToSvgApi} from '../../../../store/reducers/planToSvg';
+import type {QueryPlan, ScriptPlan} from '../../../../types/api/query';
+
+import i18n from './i18n';
+
+function getButtonView(error: string | null, isLoading: boolean) {
+    if (error) {
+        return 'flat-danger';
+    }
+    return isLoading ? 'flat-secondary' : 'flat-info';
+}
+
+interface PlanToSvgButtonProps {
+    plan: QueryPlan | ScriptPlan;
+    database: string;
+}
+
+export function PlanToSvgButton({plan, database}: PlanToSvgButtonProps) {
+    const [error, setError] = React.useState<string | null>(null);
+    const [blobUrl, setBlobUrl] = React.useState<string | null>(null);
+    const [getPlanToSvg, {isLoading}] = planToSvgApi.usePlanToSvgQueryMutation();
+
+    const handleClick = React.useCallback(() => {
+        getPlanToSvg({plan, database})
+            .unwrap()
+            .then((result) => {
+                if (blobUrl) {
```

Comment:
Lets remove this check. `blobUrl` will be revoked in cleanup function of `useEffect`.